src/ftva_etl/metadata/utils.py

```diff
@@ -66,3 +66,63 @@ def cleanup_production_type(production_type: str) -> list[str]:
     """
     # Normalize to lowercase, split by carriage returns, then strip each item's whitespace
     return [item.strip() for item in production_type.lower().split("\r")]
+
+
+def filter_alma_records(records: list, inventory_number: str) -> list:
+    """Given a list of pymarc Records from Alma as obtained via SRU, and an inventory number
+    (sourced from FTVA database), return a list of pymarc Records which match the inventory number
+    and are from the FTVA library.
+
+    :param records: A list of pymarc Records from Alma.
+    :param inventory_number: The inventory number to match.
+    :return: A list of pymarc Records from the FTVA library with matching
+    inventory number.
+    """
+
+    filtered_records = []
+    for record in records:
+        # There may be multiple AVA fields (one for each holding record), so get them all
+        fields_ava = record.get_fields("AVA")
```

Comment:
Since "AVA" is not a standard MARC21 prefix, please document what this is (Availability) and that this is embedded holdings information provided via our SRU integration, not the same as 852 holdings would normally look.